README.md

```diff
@@ -2,38 +2,31 @@
 
 A wrapper around [`htm/preact`](https://github.com/developit/htm#usage) that exposes components for building reactive HTML widgets in the context of diagram-js (core) + extensions.
 
-
 ## Getting started
+
 ```sh
 # install all dependencies
 npm install
 
-# lint and build the library 
+# lint and build the library
 npm run all
 ```
 
 ## Usage
-The module can be injected into other modules. It allows to render a Preact component and provides other utilities such as an html wrapper and preact hooks.
 
-```javascript 
-function CustomComponent(diagramJsUi) {
-  const { useState } = diagramJsUi.getHooks();
+This module is available through `diagram-js`. It allows to render a Preact component and provides other utilities such as an html wrapper and preact hooks.
+
+```javascript
+import { render, html, getHooks } form "diagram-js/lib/ui";
+
+function CustomComponent() {
+  const { useState } = getHooks();
   const counter = useState(0);
 
-  diagramJsUi.render(
-    diagramJsUi.html`
-      <div>Counter: ${counter}</div>
-    `
-  )
+  render(html`<div>Counter: ${counter}</div>`)
```

Comment:
I suggest we add a full fledged example (covering two files), cf. https://miro.com/app/board/uXjVPFGl7Kk=/?moveToWidget=3458764538136766002&cot=14.